tracim/migration/versions/da12239d9da0_delete_content_view.py

```diff
@@ -0,0 +1,187 @@
+"""delete_content_view
+
+Revision ID: da12239d9da0
+Revises: b73e57760b36
+Create Date: 2016-03-04 15:59:05.828757
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'da12239d9da0'
+down_revision = 'b73e57760b36'
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+def set_field_where_null(field_name, value="''"):
+    op.execute("UPDATE content_revisions SET %s = %s WHERE %s IS NULL" % (field_name, value, field_name))
+
+
+def set_field_to_null_where_empty_string(field_name):
+    op.execute("UPDATE content_revisions SET %s = NULL WHERE %s = ''" % (field_name, field_name))
+
+fields_names_to_empty_string = ('file_mimetype', 'file_name', 'label', 'properties',
+                                'revision_type', 'status', 'description', 'label')
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+
+    # Drop triggers
+    op.execute("DROP TRIGGER trg__contents__on_insert__set_created ON content_revisions")
+    op.execute("DROP TRIGGER trg__contents__on_update__set_updated ON content_revisions")
+    op.execute("DROP TRIGGER trg__contents__on_update On contents")
+    op.execute("DROP TRIGGER trg__workspaces__on_insert__set_created ON workspaces")
+    op.execute("DROP TRIGGER trg__workspaces__on_update__set_updated ON workspaces")
+    op.execute("DROP VIEW contents")
+
+    # Set empty string on future non null fields
+    for field_name in fields_names_to_empty_string:
+        set_field_where_null(field_name)
+
+    op.create_table('content',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.PrimaryKeyConstraint('id', name=op.f('pk__content'))
+                    )
+
+    # Create contents and reinit auto increment
+    op.execute("INSERT INTO content (id) SELECT DISTINCT(content_id) FROM content_revisions;")
+    op.execute("select setval('content_id_seq', (select max(id)+1 from content), false)")
```

Comment:
Re-setting the count of id auto-increment. I don't know why but i needed to it manually. If not next insert in ``content`` table tried to use ``1`` as ``id``.